packages/admin/admin/src/form/fields/CheckboxListField.tsx

```diff
@@ -24,7 +24,6 @@ export const CheckboxListField = <Value extends string>({ options, layout = "row
                             key={option.value}
                             label={option.label}
                             value={option.value}
-                            disabled={option.disabled}
```

Comment:
Is it really okay to remove this? Shouldn't the label also be disabled?

Reply:
Oh yeah true